augly/video/helpers/ffmpeg.py

```diff
@@ -28,11 +30,42 @@ def combine_frames_and_audio_to_file(
             "a directory"
         )
 
-    video_stream = ffmpeg.input(raw_frames, pattern_type="glob", framerate=framerate)
-    video_stream = video_stream.filter(
-        "pad", **{"width": "ceil(iw/2)*2", "height": "ceil(ih/2)*2"}
-    )
-    merge_video_and_audio(video_stream, audio, output_path)
+    with tempfile.TemporaryDirectory() as tmpdir:
+        temp_video_path = os.path.join(tmpdir, "out.mp4")
+        writer = WriteGear(output_filename=temp_video_path, logging=True)
```

Comment:
It looks like this line and the lines `writer.execute_ffmpeg_cmd()` & `writer.close()` are duplicated many times in this file. Want to make a helper function (also in this file) like `def execute_vidgear_command(output_path: str, ffmpeg_command: List[str]) -> None`? Other than that this file looks good :)

Reply:
Wouldn't the writer object need to be passed in the helper function as well if we're planning to use it in all the instances of `writer.execute_ffmpeg_cmd()` & `writer.close()`? In that case how would I identify the type for the writer object?

Reply:
You can define the `writer` object in the helper function too :) Since this is one of the duplicated lines (line 35, `writer = WriteGear ...`)